src/core/Util.js

```diff
@@ -176,7 +170,5 @@ export function requestAnimFrame(fn, context) {
 // @function cancelAnimFrame(id: Number): undefined
 // Cancels a previous `requestAnimFrame`. See also [window.cancelAnimationFrame](https://developer.mozilla.org/docs/Web/API/window/cancelAnimationFrame).
 export function cancelAnimFrame(id) {
-	if (id) {

```

Comment:
Any particular reason we are no longer checking the `id` here?

Reply:
It seems redundant — e.g. `cancelAnimationFrame(null)` is still a noop, and `requestAnimationFrame` per the spec returns a non-zero ID value.